<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>2</y>
      <width>1011</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
border-color: rgb(0, 0, 0);
background-color: rgb(181, 206, 227);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1011</width>
       <height>767</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Page1</string>
     </attribute>
     <widget class="QGroupBox" name="StateCityGroup">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>311</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font:11pt &quot;Lucida Sans&quot;;</string>
      </property>
      <property name="title">
       <string>State / City</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Zipcode</string>
       </property>
      </widget>
      <widget class="QListWidget" name="zipList">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>200</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="searchButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>290</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clearButton">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>290</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 85, 0);</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="BusinessGroup">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>0</y>
        <width>661</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font:11pt &quot;Lucida Sans&quot;;</string>
      </property>
      <property name="title">
       <string>Businesses</string>
      </property>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>661</width>
         <height>341</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QListWidget" name="categoryList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>460</y>
         <width>661</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>440</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Filter by Category</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>651</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Business Name, Street, City, Review Rating, Review Count, Num Checkins</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="popularBusinessTable">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>620</y>
        <width>311</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QTableWidget" name="succesfulBusinessTable">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>620</y>
        <width>301</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>580</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Popular Business in Zipcode:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>580</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Succesful Business in Zipcode:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="StateCityGroup_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>311</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font:11pt &quot;Lucida Sans&quot;;</string>
      </property>
      <property name="title">
       <string>Zipcode Info</string>
      </property>
      <widget class="QListWidget" name="numBusinessDisplay">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of Businesses</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Total population</string>
       </property>
      </widget>
      <widget class="QListWidget" name="totalPopulationDisplay">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>70</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Average Income</string>
       </property>
      </widget>
      <widget class="QListWidget" name="averageIncomeDisplay">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="topCatTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>271</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Top Categories:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name, #Businesses</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>600</y>
        <width>341</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Business Name, Review Rating, #Reviews</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>600</y>
        <width>341</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Business Name, #Reviews, #Checkins</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1011</width>
       <height>767</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Page 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
